src/servers/FtpServer.cc

```diff
@@ -994,8 +998,10 @@ Ftp::Server::wroteReplyData(const CommIoCbParams &io)
         return;
     }
 
-    assert(pipeline.front()->http);
-    pipeline.front()->http->out.size += io.size;
+    const auto http = pipeline.front()->http;
+    assert(http);
+    http->out.size += io.size;
+    WrittenToClient(http->al, io.size, false);
```

Comment:
This (and 2 cases below) adds statCounter.client_http.kbytes_out statistics for FTP.